astronomer/providers/google/cloud/triggers/gcs.py

```diff
@@ -87,3 +87,77 @@ async def _object_exists(self, hook: GCSHookAsync, bucket_name: str, object_name
             else:
                 res = "pending"
             return res
+
+
+class GCSPrefixBlobTrigger(GCSBlobTrigger):
+    """
+    A trigger that fires and it finds the requested file or folder present in the given bucket.
+
+    :param bucket: the bucket in the google cloud storage where the objects are residing.
+    :param object_name: the file or folder present in the bucket
```

Comment:
Check and correct the doc string and set the context of what are you doing in this trigger

Reply:
modified doc string